tests/core/failover/task.yaml

```diff
@@ -61,21 +89,41 @@ execute: |
         truncate -s 0 "$BUILD_DIR/unpack/initrd.img"
     }
 
+    if os.query is-core18 && [[ "$SPREAD_VARIANT" = "rclocalcrash" ]]; then
+        # there is no /etc/rc.local on core18
+        echo "scenario isn't supported on core18"
+        exit 0
+    fi
+
     #shellcheck source=tests/lib/names.sh
     . "$TESTSLIB"/names.sh
     #shellcheck source=tests/lib/snaps.sh
     . "$TESTSLIB"/snaps.sh
+    #shellcheck source=tests/lib/store.sh
+    . "$TESTSLIB"/store.sh
+
+    core_name="core"
+    core_snap_id="$SNAP_ID_core"
```

Comment:
We can assign TARGET_SNAP_NAME and SNAP_ID here and replace it just in case the target snap is kernel